src/AxisManager.ts

```diff
@@ -7,9 +7,9 @@ export interface Axis {
 }
 
 export interface AxisOption {
-	range?: number[];
-	bounce?: number | number[];
-	circular?: boolean | boolean[];
+	range?: [number, number];
+	bounce?: number | [number, number];
+	circular?: boolean | [boolean, boolean];
```

Comment:
Since pair of numbers at range and bounce are being used differently, It can be unnecessary for defining new type for pair of numbers to resolve this situation like `const range:(some pair type) = [0, 0]`. Therefore, this part of code might be changed back to before.